packages/GoogleAnalytics/package.json

```diff
@@ -21,7 +21,7 @@
     "setupFiles": ["../../setupTests"]
   },
   "dependencies": {
-    "@onaio/react-utils": "^0.0.1",
+    "@onaio/react-utils": "^0.0.0",
```

Comment:
Shall we have to change this to `"@onaio/react-utils": "^0.0.1",` after publishing?

Reply:
@kelvin-muchiri yes I will do that.  Had to have it like that for tests to pass.

Reply:
@moshthepitt Seems good to me

Reply:
@moshthepitt will the next publishes be affected by this?

Reply:
@ciremusyoka the next publishing should normalize things.

Reply:
@ciremusyoka, In this case, its because the new `GoogleAnalytics` package is relying on another new package `react-utils` that I added. i.e `react-utils` has not been published. I think unless a situation like this occurs again